src/XMakeBuildEngine/Evaluation/Evaluator.cs

```diff
@@ -1180,6 +1180,34 @@ private void UpdateDefaultTargets(ProjectRootElement currentProjectOrImport)
             }
         }
 
+        private string ResolveSdk(string sdkSpec, IElementLocation location)
+        {
+            string sdkName;
+            string sdkVersion;
+
+            string[] elements = sdkSpec.Split(new char[] { '/' }, 2);
+
+            //  TODO: Add error message text for InvalidSdkAttribute
+            ProjectErrorUtilities.VerifyThrowInvalidProject(elements.Length == 2, location, "InvalidSdkAttribute");
+
+            sdkName = elements[0];
+            sdkVersion = elements[1];
+
+            var sdksPath =_data.GetProperty(ReservedPropertyNames.msbuildSdksPath);
+            //  TODO: Add error message text.  Also verify whether this should actually be an "Invalid Project" exception - probably not
+            ProjectErrorUtilities.VerifyThrowInvalidProject(sdksPath != null, location, "PropertyMustBeSpecified", ReservedPropertyNames.msbuildSdksPath);
+
+            var sdkPath = Path.Combine(sdksPath.EvaluatedValue, sdkName, sdkVersion, "Sdk");
```

Comment:
Right now we don't have a `realSdkVersionFromResolutionCode`.  So how do we pick what version folder to use for the Sdk?  IE we plan to ship the .NET SDK under something like `$(MSBuildSDKsPath)\Microsoft.Net.Sdk\1.0.0-RC2`.  How would MSBuild know to look under the `1.0.0-RC2` folder without getting it from the `Sdk` attribute in the project file?